benches/benches/bevy_ecs/commands.rs

```diff
@@ -76,6 +79,58 @@ fn spawn_commands(criterion: &mut Criterion) {
     group.finish();
 }
 
+#[derive(Default)]
+struct Matrix([[f32; 4]; 4]);
+
+#[derive(Default)]
+struct Vec3([f32; 3]);
+
+fn insert_commands(criterion: &mut Criterion) {
```

Comment:
I don't love this as a name: it reads as "the time it takes to insert Commands to the queue", not "how long does it take to insert components via commands". Perhaps `batch_component_insertion`?

Reply:
`batch_component_insertion` only applies to on of the two benchmarks in the group. It is intended to be a comparison of the various insertion approaches (non batched and batched).

Reply:
I'm open to alternative names though (provided they describe that situation).

Reply:
IMO this should be `component_insertion` then. Eventually, I'd like to have non-Commands component insertion, which would also be very nice to include here. The fact that it uses `commands` to do insertion is mostly incidental IMO.

Reply:
Its worth calling out that this file is called "commands", which scopes it to "command benchmarks" imo. Happy to reorganize, but currently these benchmarks are all organized under a "commands" bucket.